README.md

```diff
@@ -1,106 +1,103 @@
 
 <pre>
-├── <a href="./services/">services</a>: Specular L2 clients
-│   ├── <a href="./services/cl_clients">cl_clients</a>: Consensus layer clients
-│   ├── <a href="./services/el_clients/">el_clients</a>: Execution layer clients
+├── <a href="./services/">services</a>: L2 services
+│   ├── <a href="./services/cl_clients">cl_clients</a>: Consensus-layer clients
+│   ├── <a href="./services/el_clients/">el_clients</a>: Execution-layer clients
 │   │      └── <a href="./services/el_clients/go-ethereum/">go-ethereum</a>: Minimally modified geth fork
-│   └── <a href="./services/sidecar/">sidecar</a>: The Specular sidecar service
-├── <a href="./contracts">contracts</a>: Specular L1 and L2 contracts
+│   └── <a href="./services/sidecar/">sidecar</a>: Sidecar services
+├── <a href="./contracts">contracts</a>: L1 and L2 contracts
 └── <a href="./lib/">lib</a>: Libraries used in L2 EL Clients
-    └── <a href="./lib/el_golang_lib/">el_golang_lib</a>: Library for golang clients
+    └── <a href="./lib/el_golang_lib/">el_golang_lib</a>: Library for golang EL clients
 </pre>
 
 ## License
 
 Unless specified in subdirectories, this repository is licensed under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0). See `LICENSE` for details.
 
-## Running a local network
-
-This guide will demonstrate how to set up a rollup network containing L2 sequencer nodes, running over a Geth L1 node---all on your local machine.
-After the 3 nodes are running, you can use MetaMask to send custom transactions to the sequencer, and see how transactions are executed on the L2 network, sequenced to the L1 network, and confirmed.
-In this example, all nodes operate honestly (no challenges are issued).
-
-### Build
-Install all dependencies and build the modified L2 Geth node
-
+## Build from source
 ```sh
-cd SPECULAR_REPO
-pnpm install
-make geth sidecar
+git clone https://github.com/specularl2/specular
+cd specular
+git submodule update --init
 ```
 
-### Generate the genesis file
-
-```sh
-SPECULAR_REPO/sbin/create_genesis.sh
+Install all dependencies and build the node binaries.
+Note: the rest of the commands below assume you are in the project root directory.
 ```
-
-### L2 setup
-
-```sh
-SPECULAR_REPO/sbin/init_geth.sh
+pnpm install
+make
 ```
 
-### L1 local dev node installation
+## Running a local devnet
 
-See [here](https://github.com/SpecularL2/specular/tree/main/contracts) for more details.
+This guide will walk you through how to set up a local devnet containing an L2 sequencer running over a local L1 node.
 
-### Start nodes
+### Configure network
 
+To configure a local devnet, you can just use an existing example from `config` as-is.
 ```sh
-# Terminal #1: start L1 node
-SPECULAR_REPO/sbin/start_l1.sh
-
-# Terminal #2: start L2 EL client
-SPECULAR_REPO/sbin/start_geth.sh
-
-# Terminal #3: start Specular sidecar
-SPECULAR_REPO/sbin/start_sidecar.sh
+# Copy all config files
+cp -a config/deployments/local_devnet/. .
```

Comment:
use a separate folder e.g. `deployments` instead of directing ppl to deploy at root dir